editor/src/components/canvas/canvas-strategies/strategies/shared-move-strategies-helpers.ts

```diff
@@ -274,7 +274,7 @@ export function getMoveCommandsForSelectedElement(
     MetadataUtils.getFrameInCanvasCoords(selectedElement, startingMetadata),
   )
 
-  const localFrame = nullIfInfinity(MetadataUtils.getFrame(selectedElement, startingMetadata))
```

Comment:
this is odd! was there a getFrame on master?

Reply:
aa I see there was, and it returned the localFrame. makes sense

Reply:
Ah yes, that's why it was named the same in this branch.